cluster/manifests/dashboard/scraper.yaml

```diff
@@ -23,28 +23,28 @@ metadata:
   name: dashboard-metrics-scraper
   namespace: kube-system
   labels:
-    application: kubernetes-dashboard
-    component: metrics-scraper
-    version: v1.0.4
+    application: kubernetes
+    component: dashboard-metrics-scraper
+    version: v1.0.7
 spec:
   replicas: 1
   selector:
     matchLabels:
-      application: kubernetes-dashboard
-      component: metrics-scraper
+      application: kubernetes
+      component: dashboard-metrics-scraper
```

Comment:
let's use `deplotyment: dashboard-metrics-scraper` as the `matchLabels` like we did for the other consolidated components.